asreview/webapp/api.py

```diff
@@ -838,6 +842,19 @@ def export_results(project_id):
             download_name=f"asreview_result_{project_id}.xlsx",
             max_age=0)
 
+    # RIS
+    else:
+
+        dataset_str = export_to_string(project_id, export_type="ris")
+
+        return Response(
+            dataset_str,
+            mimetype="application/octet-stream",
+            headers={
+                "Content-disposition":
+                f"attachment; filename=asreview_result_{project_id}.ris"
+            })
+
```

Comment:
Let's wrap this in "elif" and raise error on "else".

Reply:
I added a TypeError as else.
<img width="1338" alt="Screenshot 2021-08-09 at 16 01 19" src="https://user-images.githubusercontent.com/15941300/128719062-2b224822-c600-4e67-ba28-bceb9e6c3446.png">